src/proto/h1/dispatch.rs

```diff
@@ -519,17 +509,6 @@ cfg_server! {
             Ok(())
         }
 
-        fn poll_ready(&mut self, cx: &mut task::Context<'_>) -> Poll<Result<(), ()>> {
-            if self.in_flight.is_some() {
-                Poll::Pending
-            } else {
-                self.service.poll_ready(cx).map_err(|_e| {
```

Comment:
This would require `poll_ready` on `Server` and thus `Service`.

Reply:
Yes, I believe the simplified version is what should be kept.

Reply:
I'll do that.